git-gutter.el

```diff
@@ -227,14 +228,22 @@ gutter information of other windows."
     (with-temp-buffer
       (when (zerop (git-gutter:execute-command "git" t "rev-parse" "--is-inside-work-tree"))
         (goto-char (point-min))
-        (string= "true" (buffer-substring-no-properties
-                         (point) (line-end-position)))))))
+        (when (string= "true" (buffer-substring-no-properties
+                               (point) (line-end-position)))
+          (with-temp-buffer
+            (git-gutter:execute-command "git" t "rev-parse" "--show-toplevel")
+            (goto-char (point-max))
+            (delete-backward-char 1)
+            (buffer-string)))))))
 
 (defun git-gutter:in-repository-common-p (cmd check-subcmd repodir)
```

Comment:
This change does the same as above, returning the location of the vcs root directory (*not* the `.svn`/`.hg`/etc folder, just the directory containing it) or `nil` instead of just a boolean.